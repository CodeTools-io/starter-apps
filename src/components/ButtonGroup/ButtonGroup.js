import React, { useContext } from 'react';
import { Box, Button, ThemeContext } from 'grommet';

export default function ButtonGroup({
  buttons,
  pad = {
    vertical: 'xsmall',
    horizontal: 'medium',
  },
  ...props
}) {
  const theme = useContext(ThemeContext);

  return (
    <Box className="ButtonGroup" direction="row" align="center" {...props}>
      React.Children.map(children, function () {})
    </Box>
  );
}
