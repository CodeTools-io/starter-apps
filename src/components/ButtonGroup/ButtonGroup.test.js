import React from 'react'
import ButtonGroup from './ButtonGroup'

describe('ButtonGroup', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <ButtonGroup>
        <p>Content for the ButtonGroup</p>
      </ButtonGroup>
    )

    expect(container).toMatchSnapshot()
  })
})

