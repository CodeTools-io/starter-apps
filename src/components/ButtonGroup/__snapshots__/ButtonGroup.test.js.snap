// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 rpWqh ButtonGroup"
  >
    React.Children.map(children, function () 
    )
  </div>
</div>
`;
